seqr/migrations/0054_alter_sample_dataset_type.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.2.18 on 2023-09-15 14:54
+
+from django.db import migrations, models
+
+OLD_DT = 'SNV_INDEL'
```

Comment:
Are `OLD_DT` and `NEW_DT` meant to be the same?

Reply:
🤦 no, there was an overzealous refactor that I failed to fully undo. I tested it locally with them being different, and then later this got changed, I will change it back